src/lib/radio/radio.html

```diff
@@ -1,6 +1,7 @@
 <!-- TODO(jelbourn): render the radio on either side of the content -->
 <!-- TODO(mtlin): Evaluate trade-offs of using native radio vs. cost of additional bindings. -->
-<label [attr.for]="inputId" class="md-radio-label">
+<!-- TODO(crisbeto): only show focus indication if focus came from a keyboard event -->
+<label [attr.for]="inputId" class="md-radio-label" [class.md-ripple-focused]="_isFocused">
```

Comment:
I pulled down the PR and tested it out; I found that changing the selected radio within a group somewhat quickly (such as when using the keyboard) leaves a trailing grey circle on the radio.